test-data/de_DE.po

```diff
@@ -81,26 +81,32 @@ msgid "<HTML> is unescaped when extracted"
 msgstr "<HTML> wird unescaped extracted"
 
 #: test-data/example.jsx:119
-msgid "xxx foo bar{test}{x} y{/x} moo"
-msgstr "xxx foo bar{test}{x} y{/x} muh"
+msgid "xxx foo bar{test}{x}y{/x} moo"
+msgstr "xxx foo bar{test}{x}y{/x} muh"
 
-#: test-data/example.jsx:136
+#: test-data/example.jsx:139
 msgid "Little cats:"
 msgstr "Junge Katzen:"
 
-#: test-data/example.jsx:138
+#: test-data/example.jsx:141
 msgid "Little dogs:"
 msgstr "Junge Hunde:"
 
-#: test-data/example.jsx:140
+#: test-data/example.jsx:143
 msgid "This is an emphasized dynamic value: {number}"
 msgstr "Das ist ein hervorgehobener dynamischer Wert: {number}"
 
-#: test-data/example.jsx:144
+#: test-data/example.jsx:147
 msgid "This is an emphasized translated value: {tag}hello{/tag}"
 msgstr "Das ist ein hervorgehobener übersetzter Wert: {tag}hallo{/tag}"
 
-#: test-data/example.jsx:41 test-data/example.jsx:136
+#: test-data/example.jsx:153
+msgid "This param is using string literals: {emphasize}hello world{/emphasize}"
+msgstr ""
+"Dieser Parameter verwendet String Literals: {emphasize}Hallo "
+"Welt{/emphasize}"
```

Comment:
why do you wrap here in this way?

Reply:
that was done automatically by pybabel when updating the po file from the pot file